README.md

```diff
@@ -1,44 +1,43 @@
-modsec_parser
-=============
+[![Latest Version](https://img.shields.io/pypi/v/msc-pyparser.svg)](https://pypi.python.org/pypi/msc-pyparser)
+[![License](https://img.shields.io/badge/License-GPLv3-green.svg)](https://pypi.python.org/pypi/msc-pyparser)
+
+# modsec_parser
 
 Welcome to the `modsec_parser` documentation.
 
-Prerequisites
-=============
+The parser runs under Python 3.6+ on Linux, Windows and Mac.
+
+## Installation
+---------------
```

Comment:
May be we can drop at least `---------------` or the `##`, and use `===============`. But the rendered content is a bit weird with two underlines :).

Reply:
Changed back to `=====`